pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -96,19 +96,11 @@ def run(self):
                         outMessage = "Telegram bot setup not yet complete (master = null). Please enter your userid {} into bot configuration file.".format(update.message.from_user.id)
                         self.bot.logger.warn(outMessage)
                         continue
-                    if self.master not in [update.message.from_user.id, "@{}".format(update.message.from_user.username)]:
```

Comment:
@askovpen after #5182 is merged, uid-username relation will persist bot restarts. Also, I think removing a feature deserves a specific mentioning in the PR head, not just a line note when someone notices it.